tests/pytorch/test_numerics.py

```diff
@@ -1544,7 +1573,7 @@ def _generate_random_numbers(n, total_sum):
 @pytest.mark.parametrize("dtype", param_types)
 @pytest.mark.parametrize("num_gemms", [3, 6])
 @pytest.mark.parametrize("bs", batch_sizes)
-@pytest.mark.parametrize("model", model_configs.keys())
+@pytest.mark.parametrize("model", "126m")
```

Comment:
```suggestion
@pytest.mark.parametrize("model", ["126m"])
```
There are a few other places this is needed.